frameworks/Swift/vapor/Sources/App/Controllers/PlainTextController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  PlainTextController.swift
+//  App
+//
+//  Created by Gopal Sharma on 7/29/18.
+//
+
+import Vapor
+
+final class PlainTextController {
+
+    func get(_ req: Request) throws -> String {
```

Comment:
@ScottRobbins I used `String` instead of `StaticString` here because I believe in a "realistic" scenario, you likely wouldn't be able to always use `StaticString`.

Reply:
Sounds reasonable to me, the compiler will hopefully optimize for that anyway